virtualbox/resource_natnetwork.go

```diff
@@ -0,0 +1,94 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I am absolutely not a lawyer, so I have no idea how any of this works, but this repository is licences under MIT, Can we just pull in another licence for a single file?

Reply:
That was copied from provider.go
Feel free to re-license under MIT.